ArduCopter/control_guided_nogps.cpp

```diff
@@ -0,0 +1,30 @@
+/// -*- tab-width: 4; Mode: C++; c-basic-offset: 4; indent-tabs-mode: nil -*-
+
+#include "Copter.h"
+
+/*
+ * Init and run calls for guided_nogps flight mode
+ */
+
+// initialise guided_nogps controller
+bool Copter::guided_nogps_init(bool ignore_checks)
+{
+    if (ignore_checks) {
+        // initialise yaw
+        set_auto_yaw_mode(get_default_auto_yaw_mode(false));
+        // start in angle control mode
+        guided_angle_control_start();
+        return true;
```

Comment:
@OXINARF, thanks for that.  You're right that it might not initialise guided mode into it's angle-controller as it is, thanks!